mlflow/store/file_store.py

```diff
@@ -533,3 +535,21 @@ def _overwrite_run_info(self, run_info):
         run_dir = self._get_run_dir(run_info.experiment_id, run_info.run_uuid)
         run_info_dict = _make_persisted_run_info_dict(run_info)
         write_yaml(run_dir, FileStore.META_DATA_FILE_NAME, run_info_dict, overwrite=True)
+
+    def log_batch(self, run_uuid, metrics, params, tags):
+        _validate_run_id(run_uuid)
+        run = self.get_run(run_uuid)
+        check_run_is_active(run.info)
+        _validate_batch_log_limits(metrics, params, tags)
+        _validate_batch_log_data(metrics, params, tags)
+        try:
+            for param in params:
+                self.log_param(run_uuid, param)
+            for metric in metrics:
+                self.log_metric(run_uuid, metric)
+            for tag in tags:
+                self.set_tag(run_uuid, tag)
+        # TODO: disallow overwriting params & handle that case here similarly to in the SQLAlchemy
```

Comment:
Q: should we disallow param-overwrites in the FileStore in this PR? Or allow for the behavior difference & address it later (by definition easier to do the latter)

Reply:
Seems fine to address later.

Reply:
After all, it'd be great to just deprecate file store.